cmd/lncli/commands.go

```diff
@@ -111,13 +111,45 @@ var SendCoinsCommand = cli.Command{
 }
 
 func sendCoins(ctx *cli.Context) error {
+	var addr string = ""
+	var amt int64 = 0
+	var args cli.Args = ctx.Args()
+
+	if ctx.NArg() == 0 && ctx.NumFlags() == 0 {
+		cli.ShowCommandHelp(ctx, "sendcoins")
+		return nil
+	}
+
+	if ctx.IsSet("addr") {
+		addr = ctx.String("addr")
+	} else if args.Present() {
+		addr = args.First()
+		args = args.Tail()
+	} else {
+		return fmt.Errorf("Address argument missing")
+	}
+
+	var err error
+	if ctx.IsSet("amt") {
+		amt = int64(ctx.Int("amt"))
+	} else if args.Present() {
+		amt, err = strconv.ParseInt(args.First(), 10, 64)
+		args = args.Tail()
+	} else {
+		return fmt.Errorf("Amount argument missing")
+	}
+
+	if err != nil {
```

Comment:
Rather than checking the error generated above down here, it should instead be moved up to the point of calling.